doc/classes/CollisionObject2D.xml

```diff
@@ -31,20 +32,22 @@
 				Creates a new shape owner for the given object. Returns [code]owner_id[/code] of the new owner for future reference.
 			</description>
 		</method>
-		<method name="get_rid" qualifiers="const">
-			<return type="RID">
+		<method name="enable_shape_owner">
```

Comment:
It would be a bit clearer to call this method `set_shape_owner_enabled` since it can be used to disable as well. It can be a simple setter, without the default value.